includes/widgets/testOne.php

```diff
@@ -0,0 +1,289 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+use Elementor\Core\Kits\Documents\Tabs\Global_Colors;
+use Elementor\Core\Kits\Documents\Tabs\Global_Typography;
+
+/**
+ * Elementor testOne widget.
+ *
+ * Elementor widget that displays an eye-catching headlines.
+ *
+ * @since 1.0.0
+ */
+class Widget_testOne extends Widget_Base {
```

Comment:
@Mchris-hub Follow structure of the exisiting code.

```suggestion
class Widget_Test_One extends Widget_Base {
```

Reply:
I replaced it by widget_table, is it okay?

Reply:
@Mchris-hub Please check how it is done in other places.
I believe it should be Widget_Table.

It is important to follow coding standards.

Reply:
I changed it to widget_Table